Gemfile.lock.release

```diff
@@ -1491,7 +1490,7 @@ DEPENDENCIES
   rdoc
   rest-client (~> 2.1.0)
   routes_lazy_routes
-  rspec-rails (~> 4.0.1)
+  rspec-rails (~> 7.0)
```

Comment:
These last few are the major version bumps I can see beyond our own gems.

brakeman, factory_bot, fast_gettext, and rspec-rails.  :shipit: